lib/cleanup_helper.rb

```diff
@@ -50,14 +50,18 @@ def clean_puppet
         ActiveRecord::Base.connection.remove_column :hostgroups, :environment_id
       end
       ActiveRecord::Base.connection.drop_table(:environments, if_exists: true, force: :cascade)
+      ActiveRecord::Base.connection.drop_table(:hostgroup_puppet_facets, if_exists: true, force: :cascade)
+      ActiveRecord::Base.connection.drop_table(:host_puppet_facets, if_exists: true, force: :cascade)
 
       ActiveRecord::SchemaMigration.
         where(version: %w[20090722141107 20090802062223 20110412103238 20110712070522
                           20120824142048 20120905095532 20121018152459 20130725081334
                           20140318153157 20140407161817 20140407162007 20140407162059
                           20140413123650 20140415032811 20141109131448 20150614171717
                           20160307120453 20160626085636 20180831115634 20181023112532
-                          20181224174419]).delete_all
```

Comment:
I tried to find these migrations, but I can't find them as files. The lines were added in f0b19a28ff0bf4a67ffbaabe2bec0306e4cd5fdd. Actual files were dropped in 8c3d16e0a6dd890c88c79b4ab51dc8298f7fef64 & a18a518d09cba6037d18e6bd694d310eed0886aa.

Dropping these doesn't hurt, but since they wouldn't be recreated.